test/integration/objectstore/test_plug_media.py

```diff
@@ -309,7 +310,7 @@ def setUp(self):
         super(DataDistributionAcrossUserAndInstanceWideMedia, self).setUp()
         self.dataset_populator = DatasetPopulator(self.galaxy_interactor)
 
-    def test_media_selection_based_on_dataset_size(self):
+    def t__est_media_selection_based_on_dataset_size(self):
```

Comment:
this does not seem right

Reply:
Yes, I guess that was committed unintentionally, it will be fixed in subsequent commits.